variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,98 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+Cost of Demand-not-served|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit: €/MWh
+
+Demand-not-served|Electricity:
+   description: Demand-not-served per region (i.e. aggregate-regions, countries, nuts1, nuts2, nuts3, or nodes).
+   unit: MWh
 
 Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+   description: demand in frequency containment reserve (FCR)
    unit: MW
-   
+
 Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+   description: demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
+
 Network|Electricity|Demand|Inertia:
-   description: demand in inertia            
+   description: demand in inertia
    unit: second
-   
-Maximum Flow|Electricity|Grid:
-   description: maximum flow of an interconnection between 2 regions
-   unit: MW
-   
-Minimum Flow|Electricity|Grid:
-   description: minimum flow of an interconnection between 2 regions
+
+# description of variables related to transmission lines
+Active Power Flow|Electricity|Transmission|Line:
```

Comment:
I dont understand why this variable contains transmission|line instead of only transmission. What is the difference? If this is meant to state this is a variable related to a physical line and not the interconnection capacity between 2 regions, I would have thought this would be obvious from the value of the "region" field (if in region you have eg DE > FR, means it is transmission between "regions", if you have "node x>node y", then it is a line between the 2 nodes.

Reply:
I agree with you if we don't consider other technologies than lines in transmission but with the purpose to bring the possibility to include other equipment (power transformer, HVDC converter station, etc.) in future case studies. I think that it is important to leave this example.

Reply:
OK, understood.